RELEASING.md

```diff
@@ -0,0 +1,23 @@
+## How to Release the SDK
+
+1. Make sure that all changes for the release are merged with the master branch.
+2. From the root directory of the project in the master branch, check that the following command run cleanly.
```

Comment:
From the root directory of the project in the master branch, run the following commands.